gzip/gunzip_test.go

```diff
@@ -577,9 +575,6 @@ func TestTruncatedStreams(t *testing.T) {
 			continue
 		}
 		_, err = io.Copy(ioutil.Discard, r)
-		if ferr, ok := err.(*flate.ReadError); ok {
-			err = ferr.Err
-		}
```

Comment:
What is wrong with this unwrap?

Reply:
It is deprecated and the deprecation notice states that it is not returned any more. https://github.com/klauspost/compress/blob/d693bc87c5fbde6c49b19afaecdddf3e369389c7/flate/inflate.go#L55-L65 A quick search of GitHub confirms that it is correct.